rust/protocol/src/sealed_sender.rs

```diff
@@ -821,6 +944,15 @@ mod sealed_sender_v2 {
         }
     }
 
+    /// Encrypt a slice of random bytes `input` using a shared secret derived from the ephemeral
```

Comment:
> "Encrypt or decrypt" maybe, since it's XOR?

Yes! Done.

> maybe `input` should be one too

Yes! I ended up putting a ton of other changes relating to fixed-size arrays and removal of unnecessary `Result`s into my `docs-3` branch which is intended to follow up on #287; see https://github.com/cosmicexplorer/libsignal-client/compare/docs-2...cosmicexplorer:docs-3?expand=1. I can definitely make that change here though.